static/app/components/feedback/list/feedbackListItem.tsx

```diff
@@ -71,9 +71,11 @@ const FeedbackListItem = forwardRef<HTMLDivElement, Props>(
           <Flex column style={{gridArea: 'right'}}>
             {''}
           </Flex>
-          <strong style={{gridArea: 'user'}}>
-            <FeedbackItemUsername feedbackItem={feedbackItem} detailDisplay={false} />
-          </strong>
+          <TextOverflow>
```

Comment:
Woo! I like this

Reply:
wiat, is the `TextOverflow` blocking the gridArea from being applied?

Reply:
i don't think so, looks like everything else is still aligned okay